lua/entities/gmod_wire_lever.lua

```diff
@@ -55,6 +55,25 @@ if CLIENT then
 			self.NextRBUpdate = CurTime() + 10
 			self:SetRenderBounds(self.RBMin, self.RBMax)
 		end
+
+		local isClicking = LocalPlayer():KeyDown(IN_USE) or LocalPlayer():KeyDown(IN_ATTACK)
+		if isClicking and not self.wasClicking and IsValid(self.csmodel) then
+			local rayPos = util.IntersectRayWithOBB(
+				LocalPlayer():GetShootPos(),
+				LocalPlayer():GetEyeTrace().Normal * 75,
+				self.csmodel:GetPos(),
+				self.csmodel:GetAngles(),
+				self.csmodel:OBBMins(),
```

Comment:
self.csmodel:OBBMins() - Vector(1,1,1),
				self.csmodel:OBBMaxs() + Vector(1,1,1)

Reply:
Maybe 2 inches, for that extra girth

Reply:
KeyPress hook might be better